_posts/2019-11-20-bazel-1.2.0.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: posts
+title: "Bazel 1.2"
+authors:
+  - philwo
+---
+
+We've just released [Bazel 1.2](https://github.com/bazelbuild/bazel/releases/tag/1.2.0)!
+
+Bazel 1.2 is intended to be backward-compatible with Bazel 1.0, however it contains a single incompatible change that was deemed a bug fix and thus eligible for inclusion.
+
+Please [report](https://github.com/bazelbuild/bazel/issues/new) any update problems.
+
+## Incompatible changes
+
+* Tree artifacts and regular artifact paths can no longer overlap. This could happen when a Starlark action creates a tree artifact and a file artifact underneath the tree. This triggered undefined behavior (e.g. what should a downstream action that depends on the tree artifact see?) and is now explicitly forbidden. See [d0ce12d](https://github.com/bazelbuild/bazel/commit/d0ce12d6f8140edd4644e4a2777620d3bbfbb1dc) for some details.
+
+## Android
```

Comment:
I think it would be nice to add a macOS / iOS section mentioning this bug fix https://github.com/bazelbuild/bazel/issues/10184

Something like:

```
* Fixed a bug that broke caching for all builds on macOS
```

So that users know about this issue as an incentive to upgrade

Reply:
Great idea, thank you! I've added it.